src/ProblemDetailsNotFoundHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Zend\ProblemDetails;
+
+use Interop\Http\ServerMiddleware\DelegateInterface;
+use Interop\Http\ServerMiddleware\MiddlewareInterface as ServerMiddlewareInterface;
+use Negotiation\Negotiator;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Zend\Stratigility\Delegate\CallableDelegateDecorator;
+
+class ProblemDetailsNotFoundHandler implements ServerMiddlewareInterface
+{
+    /**
+     * @var ProblemDetailsResponseFactory
+     */
+    private $responseFactory;
+
+    /**
+     * @param ProblemDetailsResponseFactory $responseFactory Factory to create a response to
```

Comment:
Type in PHPDocs should be consistent with method declaration: `null|ProblemDetailsResponseFactory`